gcc-vm.yml

```diff
@@ -0,0 +1,147 @@
+---
+- hosts: GCCVM
+  become: true
+  tasks:
+  - name: Install the 'Development tools' package group
+    yum:
+      name: "@Development tools"
+      state: latest
+      update_cache: yes
+
+  - name: Install packages required to create gcc rpm with omnibus
+    yum:
+      name: "{{ item }}"
+      state: latest
+    with_items:
+    - rpm-build
+    - glibc-devel.i686
+    - libgcc.i686
+
+  - name: Remove repository (and clean up left-over metadata)
+    yum_repository:
+      name: wandisco
+      state: absent
+
+  - name: Install wandisco git repository to install git from it
+    yum:
+      name: http://opensource.wandisco.com/centos/6/git/x86_64/wandisco-git-release-6-1.noarch.rpm
+      state: present
+
+- hosts: GCCVM
+  become: true
+  vars:
+          # created with:
+          # python -c 'import crypt; print crypt.crypt("1ns3cur3", "insecure")'
+    password: inyr/N2x97F.E
+
+  tasks:
+  - name: Install the latest version of git from wandisco repository
+    yum:
+      name: git
+      state: latest
+
+- hosts: GCCVM
+  vars:
+    ruby_version: "2.2.0"
+    rvm_path: "/home/rpm_omnibus/rvm/gems/ruby-{{ ruby_version }}/bin:/home/rpm_omnibus/rvm/gems/ruby-{{ ruby_version }}@global/bin:/home/rpm_omnibus/rvm/rubies/ruby-{{ ruby_version }}/bin:/home/rpm_omnibus/rvm/bin:/home/rpm_omnibus/gem/ruby/2.2.0/bin"
```

Comment:
Some possible improvements here:
- `2.2.0` - to be replaced with `{{ ruby_version }}`
- Some of the paths generated here, are used later, maybe you can make some variables for them too?  In fact, you are creating `ruby_gem_path` in the line below, that could be used here.